### ISSUE TITLE
Submit epic 1 datapackages into prod pipeline

### ISSUE THREAD

**Issue by karlcz** [nih-cfde/cfde-deriva#120]
After the pipeline components are deployed, we need to run the submissions for the prior (epic 1) data to populate the system and to sanity check all the provisioned elements and policies.  Appropriate members of each DCC's submitter group must run the submission tool.

Prod
- [ ] rebuild "release" catalog using bags collected in prior staging pass (Karl)
- [ ] hmp
- [ ] gtex
- [ ] motrpac
- [ ] kidsfirst
- [ ] metabolomics
- [ ] lincs
- [ ] 4dn

**Comment by karlcz** [comment]
This issue was based on the idea that the initial registry config will contain hand-built records matching "epic1" data.  But, should we instead deploy an empty pipeline and include execution of the submission tools to get the prior submissions reingested as part of the actual deployment sequence?

**Comment by karlcz** [comment]
Hmm, I have redundant issues so am going to carve out the "staging" part of this one and make this one just about production.

**Comment by karlcz** [comment]
Furthermore, should we even consider having epic1 datapackages in the pipeline as part of the epic 2 release?  Or do we want to assert that DCCs need to submit their content via the new submission tool as a prerequisite for subsequent review and eventual release?

One tricky aspect is we would not be able to build a future release until we get an approved submission for ALL constituent DCCs, so either they or CFDE-CC would have to resubmit the epic 1 content if they want to continue using it for interim releases.

**Comment by karlcz** [comment]
@lliming  @ACharbonneau @owhite   What are your opinions on this?  Do we want the DCCs to start with an "empty" list until they successfully use the submission tool themselves?  Or do we want to seed the list with their prior "epic 1" content, corresponding to what we have already ingested by other ad hoc means?

**Comment by ACharbonneau** [comment]
I'm not sure. I have a lot of feelings about this whole thing, and I think this might need to be a policy team or roadmap talk. 

For your most proximate question, I don't think we want to give them an old submission to approve, because we want them to make their own, so we can pull the 'testing' banner and make the site actually public. Right now we have advertised it to exactly no one outside the CFDE, and we really need to start having users try it, and we can't to that until the data is real.

My belief was that for epic 2, the public catalog wasn't changing, and that we were keeping the 'testing data' banner on the site. We've also said we were going to do quarterly pushes of the public catalog, which I was assuming meant that it would look as is until end of march, at which point we have a new public catalog with *only* data submitted by DCCs, and we could start publicizing a bit.

That said, that plan will dramatically change our site and what it can do, because theres a really good chance we're only really going to have level 0 submissions (except for HMP), and that we'll only have 4 DCCs.

Unless of course we start really pushing funded DCCs to make submissions *and also* start working with the unfunded DCCs who have data in now to get them started on submissions. I have not been pushing either of those, it's already taking up a ton of my time just supporting the 4 funded DCCs that are actively working on submissions. So if we're going to try to push people, it really needs to start now. 

Back to the proximate question, we could decide instead to let them keep their old submission and prepopulate their submission catalog, however if they choose that one then I don't know where that puts us on removing the testing banner and making the data public. We'd have to be really sure they knew what they were clicking, or it might turn into demands to revert and all sorts of drama.

We also haven't yet given the DCCs a timeline for when they would have to approve a submission to be included in the first real release, regardless of who made the catalog. Again, this is a thing we need to make public pretty soon or no one will meet the deadline, we'll have a release with just HMP data, etc, etc

TLDR; theres a lot of implications regardless of what we choose and I genuinely don't know which way is best.

**Comment by karlcz** [comment]
This issue is obsolete as we DO NOT plan to prepare submissions representing epic 1 state, and the release catalog task is part of the prod release checklist and so does not need this issue to track it.
